cloudman/helmsman/tests/test_helmsman_api.py

```diff
@@ -74,3 +74,44 @@ def test_crud_chart(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(response.data['results']), 1)
+
+
+class NamespaceServiceTests(HelmsManServiceTestBase):
+
+    NAMESPACE_DATA = {
+        'name': 'newnamespace',
+        'status': 'Active',
+        'age': '1d'
+    }
+
+    def test_crud_namespace(self):
+        """
+        Ensure we can register a new cluster with cloudman.
+        """
+        # create the object
+        url = reverse('helmsman:namespaces-list')
```

Comment:
Somehow, the coverage has dropped. I was hoping we could finally hit 90 with this PR. We are this close.

Reply:
Yes, some of the exceptions don't have tests. I can probably get to 90 today